src/Asynkron.JsEngine/StandardLibrary.cs

```diff
@@ -5,6 +5,22 @@ namespace Asynkron.JsEngine;
 /// </summary>
 public static class StandardLibrary
 {
+    /// <summary>
+    /// Converts a JavaScript value to its string representation, handling functions appropriately.
+    /// </summary>
+    private static string JsValueToString(object? value)
+    {
+        return value switch
+        {
+            null => "null",
+            string s => s,
```

Comment:
should this not also cover the good old [object, Object] or whatever the syntax is?